lib/config/presets/internal/monorepo.ts

```diff
@@ -492,6 +492,11 @@ const repoGroups = {
   'trust-dns': 'https://github.com/bluejekyll/trust-dns',
   tsoa: 'https://github.com/lukeautry/tsoa',
   turbo: 'https://github.com/vercel/turbo',
+  'typed-signalr': [
```

Comment:
> > Do these repos all release synchronized versions at the same time?
> 
> I'm not sure exactly what you're asking - but i don't think so. As in, if packages in the main repo are updated, this does not automatically mean a release of .DevTools and/or .Typescript, the author seems to keep things seperate in that regard. I can imagine if there are breaking changes all packages of all git repos would be updated.
> 
> Would you like me to remove those and then simply refer to the main repo?

Commenting this as a review comment so we don't accidentally merge this before it's ready. I'll let the maintainers decide what to do.